_infra/helm/securebanking-openbanking-uk-rcs/values.yaml

```diff
@@ -32,7 +32,7 @@ deployment:
       memory: 512Mi
 
 ingress:
-  apiVersion: extensions/v1beta1
+  apiVersion: networking.k8s.io/v1
```

Comment:
As a developer I don't know anything about the ingress chart.
What is the reason of the `apiVersion` update?

Reply:
The reason for the update is because the new GKE version v1.22 version does not support "extensions/v1beta1"